shared/Consts.js

```diff
@@ -36,5 +36,10 @@ export const DEFAULTS = {
         BOOST: 0.5,
         P_ROTATE: 0.1,
         N_ROTATE: -0.1
+    },
+    DRAG: {
+        rho = 1.204,
```

Comment:
There is a syntax error. In objects use `:` not `=` to aassign properties. See above object for refrence.
```suggestion
        rho: 1.204
```
also change for 2 other properties